### ISSUE TITLE
pytorch version

### ISSUE THREAD

**Issue by acht7111020** [r06922019/tensorflow-spectral-normalization#1]
Can you provide pytorch version please? :)
